changelog/new_use_packwerk_compatible_globs.md

```diff
@@ -0,0 +1 @@
+* [#234](https://github.com/rubocop/rubocop-rails/issues/234): Use packwerk-compatible globs. ([@tdeo][])
```

Comment:
Could you also add a note about Engine? e.g.,

```suggestion
* [#234](https://github.com/rubocop/rubocop-rails/issues/234): Use globs that are compatible with Engine or Packwerk. ([@tdeo][])
```

Reply:
Also, could you start the filename with `change_` instead of `new_`?

Reply:
I did fix those 2 things

Reply:
There is no new changelog file, so it may not have been added with `git add`.

Reply:
Sorry for missing that 🤦